### DISCUSSION TITLE
Start always a new Window

### DISCUSSION THREAD

**Discussion by bforpc** [zenangst/KeyboardCowboy#455] [General]
Hi,

I love KC. I work most of it to open a terminal (iterm2). But I like to open on my shortcut always a new Window, not focus the always opened.

Bfo

**Comment by zenangst** [comment]
Hey @bforpc, thanks for those lovely words. Glad you liked it.

If I understood this correctly, would something like this work?

https://github.com/zenangst/KeyboardCowboy/assets/57446/875c0d6d-e990-4122-a83e-c1707c8ae42e

If not, then could you elaborate a bit further?

Thanks!

**Comment by bforpc** [comment]
Hello zenangst,

this is not working. If i press the button "Run Serial", then - if no iterm ist open - it will be open, then press the button again and the already opened window will be fucused. No new window ist comming up.
Test in 2 Macs. One with Intel and one with M1 CPU. Both of them newest OS Updates installed.

Bfo

**Reply by zenangst** [reply]
@bforpc I came up with another solution in the issue that you posted here: https://github.com/zenangst/KeyboardCowboy/issues/491#issuecomment-1937904565

Hope that is more inline with what you want.
